inc/class-cachify-interface.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Interface for APC based caching storage classes.
+ *
+ * @package Cachify
+ */
+
+/* Quit */
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Interface_Cachify_Storage
+ */
+interface Cachify_Interface {
```

Comment:
The `Cachify_` prefix is used like a namespace substitute, so the actual interface name is `Interface`. Maybe something like `Cachify_Backend` or similar would be little more descriptive.

Reply:
Yeah, I adjusted it by iteration. After realizing there is an autoloader with a naming pattern. ;-) But yours is good input, I will go to change it.